pkg/util/membercluster_client.go

```diff
@@ -31,6 +32,12 @@ type DynamicClusterClient struct {
 	ClusterName      string
 }
 
+// AggregatorClusterClient stands for a aggregator client for the given member cluster
+type AggregatorClusterClient struct {
+	AggregatorClient *aggregatorclient.Clientset
+	ClusterName      string
+}
```

Comment:
I guess this is a discovery client, right? 
I'm not sure if we really need this, (`clusterClient.KubeClient.Discovery().ServerGroupsAndResources()` can't help? )might be @XiShanYongYe-Chang could take a look.

Reply:
It is a `aggregatorclient` for create a apiservice resource,which is needed for e2e test,not for list api-resources.

Reply:
This client appears to be used to simulate creating and getting APIService resources in a member cluster.